codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -402,6 +403,15 @@ public Http2Stream resetSent() {
         }
 
         @Override
+        public Http2Stream headersSent() {
+            headersSent = true;
+            return this;
+        }
+
+        @Override
+        public boolean isHeadersSent() { return headersSent; }
```

Comment:
Not sure I understand what you mean here. I think I follow the format of the other similar methods?